WalletWasabi.Fluent/ViewModels/CoinJoinProfiles/CoinJoinProfileViewModelBase.cs

```diff
@@ -8,7 +8,7 @@ public abstract class CoinJoinProfileViewModelBase : ViewModelBase
 
 	public abstract string Description { get; }
 
-	public abstract IImage Icon { get; }
+	public virtual IImage? Icon { get; }
```

Comment:
When can `Icon` be `null`?
What happens then?

Reply:
redundant... we no longer use this.